assets/src/story-embed-block/block/edit.css

```diff
@@ -1,5 +1,11 @@
 /* Loading Indicator */
```

Comment:
Need to move that comment to the right placce now, FWIW

Reply:
Fixed.